tests/store/model_registry/test_sqlalchemy_store.py

```diff
@@ -515,6 +515,11 @@ def test_search_registered_models(self):
         rms, _ = self._search_registered_models(f"name iLike '%blah%'")
         self.assertEqual(rms, [])
 
+        # confirm that ILIKE works for empty query
+        rms, _ = self._search_registered_models(f"name iLike '%%'")
+        # raise Exception()
```

Comment:
?

Reply:
This is the query that the UI sends when the page first loads - the comment raising Exception is something I missed while debugging. Removed now